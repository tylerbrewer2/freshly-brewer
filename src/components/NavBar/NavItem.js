import React from 'react';
import styled from 'styled-components';

const richBlack = '#4A4A4A'

const NavItem = styled.li`
  list-style: none;
  margin-left: 30px;
  float: left;
  font-size: 12px;
  color: ${richBlack}
  letter-spacing: 3px;
`

export default NavItem;
