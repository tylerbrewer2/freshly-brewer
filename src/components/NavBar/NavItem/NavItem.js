import styled from 'styled-components';
import { Link } from 'react-router-dom';

const richBlack = '#4A4A4A'

const NavItem = styled(Link)`
  font-family: 'AvenirNextBold', 'Helvetica', 'sans-serif';
  list-style: none;
  margin-left: 30px;
  float: left;
  font-size: 12px;
  color: ${richBlack};
  letter-spacing: 3px;
  text-decoration: none;
`

export default NavItem;
