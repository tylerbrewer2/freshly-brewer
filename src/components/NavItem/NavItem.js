import styled from 'styled-components';
import { Link } from 'react-router-dom';

import Fonts from '../../styles/helpers/Fonts';

const richBlack = '#4A4A4A'

const NavItem = styled(Link)`
  font-family: ${Fonts.body};
  list-style: none;
  margin-top: ${ props => props.marginTop ? props.marginTop : '85px'};
  margin-left: 45px;
  margin-right: 45px;
  float: left;
  font-size: 12px;
  color: ${richBlack};
  letter-spacing: 3px;
  text-decoration: none;
  &:hover {
    text-decoration: underline;
  }
`

export default NavItem;
