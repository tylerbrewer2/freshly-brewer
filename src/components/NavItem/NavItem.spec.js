import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import NavItem from './NavItem';

it('renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <NavItem to='test'>Test item</NavItem>
    </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
