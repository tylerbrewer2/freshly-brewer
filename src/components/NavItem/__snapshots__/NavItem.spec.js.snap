exports[`test renders correctly 1`] = `
<a
  className="sc-bdVaJa gjWQVa"
  href="test"
  onClick={[Function]}>
  Test item
</a>
`;
